src/App.tsx

```diff
@@ -67,6 +68,9 @@ const App: React.FC = () => {
         if (isSameStatus) {
           return prevStatus
         }
+        if (batches && batches[0] && !batches[0].endedAt) {
```

Comment:
Shouldn't it be:

```suggestion
        if (batches && batches[0] && batches[0].endedAt) {
```

otherwise it means it's still updating that batch?

Reply:
Yup. I was troubleshooting this as you were reviewing!